app/views/components/_summary_card_component.html.erb

```diff
@@ -0,0 +1,41 @@
+<%
+  id ||= nil
+  data_attributes ||= {}
+  summary_card_actions ||= []
+  rows ||=[]
+%>
+<%= tag.div class: "app-c-summary-card-component", id: id, data: data_attributes do %>
+  <div class="govuk-summary-card">
+    <div class="govuk-summary-card__title-wrapper">
+      <h2 class="govuk-summary-card__title"><%= title %></h2>
+      <ul class="govuk-summary-card__actions">
+        <% summary_card_actions.each do |action| %>
+          <li class="govuk-summary-card__action">
+            <%= link_to action[:label], action[:href], class: "govuk-link govuk-link--no-visited-state #{"gem-link--destructive govuk-!-font-weight-bold" if action[:destructive]}".strip %>
```

Comment:
For accessibility where we have links we should provide assistive technology with more information so that there isn't a collection of "edit" (or whatever) links with no context. Instead a screen reader would read "edit key one", "edit key two" etc. 

We can either add this with an aria-label or provide some hidden text like `<a>edit<span class="govuk-visually-hidden"> title</span></a>`